tests/features/generated_story_tables.feature

```diff
@@ -29,73 +33,69 @@ The below scenario covers story table tests for:
+    | x[i].name.first | Proxyname2 |  | proxy_list[1].name.first |
+    | radio_other | radio_other_opt_3 |  |  |
+    | radio_yesno | False | false |  |
+    | screen_features | True | true |  |
+    | showif_checkbox_yesno | False | false |  |
+    | showif_checkboxes_other | showif_checkboxes_nota_1 | false |  |
+    | showif_checkboxes_other | showif_checkboxes_nota_2 | true |  |
+    | showif_checkboxes_other | showif_checkboxes_nota_3 | false |  |
+    | showif_dropdown | showif_dropdown_1 |  |  |
+    | showif_radio_other | showif_radio_multi_2 |  |  |
+    | showif_text_input | Show if text input value |  |  |
+    | showif_textarea | Show if\nmultiline text\narea value |  |  |
+    | showif_yesnoradio | True | true |  |
+    | text_input | Regular text input field value |  |  |
+    | textarea | Multiline text\narea value |  |  |
+    | show_3 | True | true |  |
+    | show_2 | True | true |  |
+    | signature_1 | /sign |  |  |
+    | signature_2 | /sign |  |  |
 
-@generated @slow @2 @convert_original
-Scenario: Format of story table is original
+@fast @stf2
+Scenario: Table is format 1 (og three cols)
   Given I start the interview at "all_tests"
-  And the user gets to "the end" with this data:
+  And the user gets to "showifs" with this data:
     | var | choice | value |
-    | direct_showifs | True | true |
-    | button_continue | True | true |
-    | buttons_other | | button_2 |
-    | buttons_yesnomaybe | True | true |
     | checkboxes_other | checkbox_other_opt_1 | true |
-    | checkboxes_other | checkbox_other_opt_2 | true |
-    | checkboxes_other | checkbox_other_opt_3 | false |
     | checkboxes_yesno | True | true |
-    | direct_standard_fields | True | true |
     | dropdown_test | | dropdown_opt_2 |
-    | proxy_list[0].name.first |  | Proxyname1 |
-    | proxy_list[1].name.first |  | Proxyname2 |
     | radio_other | | radio_other_opt_3 |
-    | radio_yesno | False | false |
-    | screen_features | True | true |
-    | showif_checkbox_yesno | False | false |
-    | showif_checkboxes_other | showif_checkboxes_nota_1 | false |
-    | showif_checkboxes_other | showif_checkboxes_nota_2 | true |
-    | showif_checkboxes_other | showif_checkboxes_nota_3 | false |
-    | showif_dropdown | | showif_dropdown_1 |
-    | showif_radio_other | | showif_radio_multi_2 |
-    | showif_text_input |  | Show if text input value |
-    | showif_textarea |  | Show if\nmultiline text\narea value |
-    | showif_yesnoradio | True | true |
+    | radio_yesno | False | true |
     | text_input | | Regular text input field value |
     | textarea | | Multiline text\narea value |
-    | show_3 | True | true |
-    | show_2 | True | true |
-    |  |  | /sign |
+  Then I arrive at the next page
+
+# TODO: Test og table with `sought` col added?
+
+@fast @stf3
+Scenario: Table is format 2 (no sought var)
+  Given I start the interview at "all_tests"
+  And the user gets to "showifs" with this data:
+    | var | value | checked |
```

Comment:
The main columns right now are `| var | value | checked |`. In this new version, I want to combine `| value |` and `| checked |`. I think it's possible. I think the wording needs to be worked on. I think this is the next priority so that we don't have to do this yet again after a bunch of interviews start using this version in case we find that it's a breaking change.

Reply:
The key is checkboxes. Right now, you say 'this checkbox option is true (checked)' (or false for unchecked). The shift would be that the developer would only list the checkboxes that are checked and we'd make sure the other, unnamed, checkboxes were unchecked. This is a bit of a different paradigm, but one we can do now that we collect all the elements on the page. I'm not completely sure it's possible.